__tests__/options.test.ts

```diff
@@ -68,6 +72,18 @@ describe('options', () => {
     expect(getQueryParams(route)[1].required).toEqual(false)
   })
 
+  it('uses the explicit `type` parameter to override request body type', () => {
```

Comment:
Testing for query params right? Not request body as the test name says

Reply:
Done.